Sources/Player/PlaybackEngine/Internal/PlayerItemLoader.swift

```diff
@@ -36,20 +42,30 @@ final class PlayerItemLoader {
 	func renderVideo(in view: AVPlayerLayer) {
 		playerLoader.renderVideo(in: view)
 	}
+
+	func updateConfiguration(_ configuration: Configuration) {
+		self.configuration = configuration
+	}
 }
 
 private extension PlayerItemLoader {
 	func load(_ mediaProduct: MediaProduct, with streamingSessionId: String) async throws -> (Metadata, Asset) {
-		if let storedMediaProduct = mediaProduct as? StoredMediaProduct {
+		let offlinePlaybackAllowed = offlinePlaybackPrivilegeCheck?() ?? true
```

Comment:
I don't know the context, but is `true` fine as a default value?

Reply:
Resolved those suggestions in [the original PR](https://github.com/tidal-music/tidal-sdk-ios/pull/96)